platform/darwin/src/MGLStyleLayer.h

```diff
@@ -69,6 +69,17 @@ MGL_EXPORT
  */
 @property (nonatomic, assign) float minimumZoomLevel;
 
+#pragma mark Layer Transitions
+
+- (void)setTransition:(MGLTransition)transition forKey:(NSString *)key;
+
+- (MGLTransition)transitionForKey:(NSString *)key;
+
+/**
+ Returns an array of strings that identify transition keys attached to the layer.
+ */
++ (NSArray *)transitionKeys;
```

Comment:
Should we expose this method publicly? If so, let's call it `transitionableAttributes` and also make the return type a set of strings instead of an unqualified array.